### PR TITLE
added navigation

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

**Review by vivekgudipudi — APPROVED** [review]

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.navbar{
+  background-color:var(--preferred-color);
+  display:flex;
+  padding:0.7rem;
+  border-top-left-radius: 0.3rem;
+  border-top-right-radius: 0.3rem;
+
```
Comment (smkmallik) [inline components/navigation/navigation.css:7]: There's only one code refactoring change that you can do it in future is that currently you have given direct values to all the css properties like padding and border radius. Try to declare a global variable in :root and use those global variables to use in properties.

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  font-weight: 700;
+  color:var(--text-color);
+}
+.nav-pills{
+  display:flex;
+  flex-wrap:wrap;
+  color:var(--text-color);
+  margin-left:700px;
+  gap:20px;
+}
```
Comment (smkmallik) [inline components/navigation/navigation.css:21]: While saving the css file, you can prettier to format the css file, you haven't used spaces after colon. For eg. gap:20px should be gap:<space>20px

**Comment by smkmallik** [comment]
Added some minor comments to refactor the code as you develop further on this.
Everything seems good, you can merge the changes.

**Review by smkmallik — APPROVED** [review]

File: components/navigation/navigation.css (hunk 1 above)
Comment (MiryalaAkshitha) [reply components/navigation/navigation.css:7]: sure soumik! I'll try to do it!

**Comment by vivekgudipudi** [comment]
better to declare lengths in utilities and using variables wherever needed and try to use rems instead of px.
you can fix these when u get time, good to go now, UI looks nice.

**Pull request merged**